.github/workflows/android-perf.yml

```diff
@@ -157,7 +159,16 @@ jobs:
         BUILD_MODE="cmake"
         DTYPE="fp32"
 
-        if [[ ${{ matrix.model }} =~ ^stories* ]]; then
+        if [[ ${{ matrix.model }} =~ ^[^/]+/[^/]+$ ]] && [[ ${{ matrix.delegate }} == "xnnpack" ]]; then
+            pip install -U "huggingface_hub[cli]"
```

Comment:
Nit: You could consider moving line 163 to 167 to a script in https://github.com/pytorch/executorch/tree/main/.ci/docker/common, i.e. `install_huggingface.sh`, and make it available in the Docker image instead of installing it here.  Having it in the Docker image isa bit more efficient.